specs/ledger/hs/test/Ledger/Delegation/Properties.hs

```diff
@@ -211,9 +210,9 @@ expectedDms s d cs = Map.fromList (fmap (delegator &&& delegate) activeCerts)
     activeCerts = concatMap _blockCerts activeBlocks
 
 minusSlotCount :: Slot -> SlotCount -> Slot
-minusSlotCount (Slot s) (SlotCount c) = case s `minusNaturalMaybe` c of
-  Nothing -> Slot 0
-  Just k  -> Slot k
+minusSlotCount (Slot s) (SlotCount c)
```

Comment:
Can we not import this from Ledger.Core?

Reply:
seems like duplicate code

Reply:
No, because the other function subtracts a slot to a slot count, so there are different functions. I actually got some failing tests because I thought `minusSlot` was doing that `minusSlotCount` is doing :/

Reply:
Ah. So in this case it might be nice to better document these!

Reply:
What about [this](https://github.com/input-output-hk/fm-ledger-rules/pull/249/commits/3f1d5454ec4f27e3f0acc9a2d08683f60e2e40ce)